system/security/shared/caching.cpp

```diff
@@ -525,7 +526,7 @@ inline void CPermissionsCache::removeManagedFileScopes(IArrayOf<ISecResource>& s
 
 inline void CPermissionsCache::removeAllManagedFileScopes()
 {
-    WriteLockBlock writeLock(m_scopesRWLock);
+    //WriteLockBlock writeLock(m_scopesRWLock);
```

Comment:
worth changing this to a comment, stating that this method must be called while m_scopesRWLock lock held.

Reply:
Agreed